sms_toolkit/messages/normalization.py

```diff
@@ -13,14 +15,14 @@ def normalize_optimistic_gsm_characters(message):
     Returns:
         str - a normalized unicode string
     """
-    if isinstance(message, str):
+    if isinstance(message, six.binary_type):
```

Comment:
Does this also include `text_type`? I think we were tossing binary strings and normal strings in here.

Reply:
text_type is `unicode` in py 2 and `str` in py 3
binary_type is `str` in py2 and `bytes` in py3

I think the logic is that if we have a non unicode string, we should convert it to unicode. Non unicode strings are `six.binary_type`